src/os/view/header/contextMenu/navigation.tsx

```diff
@@ -15,7 +15,7 @@ const Navigation = () => {
     <Row gutter={[12, 12]} wrap={false} align="middle">
       {appIds.map((appId) => (
         <Col key={appId}>
-          <Badge dot={params?.appId === appId} color="#5D6CCF" offset={[-4, 4]}>
+          <Badge dot={params?.appId === appId} className="app-active" offset={[-4, 4]}>
```

Comment:
app-active -> sentre-app-active may be better to avoid class conflict with other apps. app-active is a pretty common class name